common/Timer.cc

```diff
@@ -1,30 +1,50 @@
 #include "common/Timer.h"
+#include "common/sort.h"
 using namespace std;
 namespace sorbet {
+namespace {
+vector<int> ensureSorted(initializer_list<int> input) {
+    vector<int> output(input);
+    // Keep empty output empty; that means that there is no histogram.
+    if (!output.empty()) {
+        // Append a catch-all bucket.
+        output.push_back(INT_MAX);
+        fast_sort(output);
```

Comment:
We want to keep timer creation as fast as possible. There are two ways this degrades that:

- `fast_sort` is a ton of code. It likely won't be inlined
- `fast_sort` might add non-trivial runtime to the thing we're measuring.

Do you think we can move this code elsewhere? Dmitry suggested in the statsd submission code.